### ISSUE TITLE
specification never finished computing

### ISSUE THREAD

**Issue by jaypantone** [PermutaTriangle/Tilings#183]
The script
```python
from tilings.tilescope import TileScope, TileScopePack

basis = '0123_0132_1230_3012'
pack = TileScopePack.insertion_row_and_col_placements(row_only=True).make_fusion(apply_first=True)
css = TileScope(basis, pack, debug=True)
spec = css.auto_search(status_update=30, perc=10)
```
leads to some kind of infinite computation in the specification creation stage, where larger and larger labels are expanded.

Edit: NOTE that this issue only occurs sometimes You may need to run the snippet above several times to recreate it.

**Comment by jaypantone** [comment]
Now I can't recreate this anymore. Never mind.

**Comment by christianbean** [comment]
This is a very strange bug - we could try rerunning with expand_verified=False to see if it makes a difference.

**Comment by jaypantone** [comment]
Actually, sorry, I can recreate it. But it's not every time.

**Comment by christianbean** [comment]
This should be fixed by #186
